src/TodoContainer.js

```diff
@@ -0,0 +1,137 @@
+import React, { useState, useCallback, useEffect } from "react";
+import TodoItem from "./components/TodoItem";
+import {
+  TodoWrapper,
+  Title,
+  CreateTodo,
+  TodoInput,
+  AllButton,
+  ActiveButton,
+  CompletedButton,
+  TodoList,
+  ClearTodo,
+} from "./styles/TodoContainer.style";
+
+export default function TodoContainer() {
+  const todoStatus = Object.freeze({
+    All: "all",
+    Done: "done",
+    Undone: "undone",
+  });
+
+  const [todoList, setTodoList] = useState([]);
+  const [todoContent, setTodoContent] = useState("");
+  const [showTodoStatus, setShowTodoStatus] = useState(todoStatus.All);
+
+  const getId = () => {
+    let id = new Date().getTime();
+    return id;
+  };
+
+  useEffect(() => {
+    const todoList = JSON.parse(localStorage.getItem("todoList"));
+    if (todoList) {
+      setTodoList(todoList);
+    }
+  }, []);
+
+  const addTodoLocalStorage = (todoList, todo) => {
+    localStorage.setItem("todoList", JSON.stringify([...todoList, todo]));
+    setTodoList([...todoList, todo]);
+  };
+
+  const reviseTodoLocalStorage = (todo) => {
+    localStorage.setItem("todoList", JSON.stringify(todo));
+    setTodoList(todo);
+  };
+
+  const handleAddTodo = useCallback(
+    (e) => {
+      if (!todoContent || todoContent.trim() === "") return;
```

Comment:
can you pass a local attribute instead of the global one in the function? you can pass `todoContent` when you call the function on line 111